transport/transport_test.go

```diff
@@ -1,9 +1,6 @@
 package transport_test
```

Comment:
With the deprecated test removed, nothing in this file is used any more (in particular, `setup`). We can keep it if you think it might get used at a later stage. In any case, this might suggest that we need to create some transport tests now that there are none.